src/audio/dai.c

```diff
@@ -691,6 +691,27 @@ static int dai_config(struct comp_dev *dev, struct sof_ipc_dai_config *config)
 	return 0;
 }
 
+static void dai_cache(struct comp_dev *dev, int cmd)
+{
+	struct dai_data *dd = comp_get_drvdata(dev);
```

Comment:
My preference here is that dau_cache() would check cmd and their either call static dai_cache_invalidate or dai_cache_writeback. This should allow for differences in sequences and by adding different trace.